src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -364,103 +454,131 @@ public void ShadowCopyFiles()
-            Assembly.LoadFile(typeof(AppDomain).Assembly.Location);
-            Assembly[] assemblies2 = AppDomain.CurrentDomain.GetAssemblies();
-            Assert.True(assemblies2.Length > assemblies.Length, "Assembly.LoadFile should cause an increase in GetAssemblies list");
-            int ctr = 0;
-            foreach (var a in assemblies2)
-            {
-                // Dynamic assemblies do not support Location property.
-                if (!a.IsDynamic)
+            RemoteInvoke(() => {
+                Assembly[] assemblies = AppDomain.CurrentDomain.GetAssemblies();
+                Assert.NotNull(assemblies);
+                Assert.True(assemblies.Length > 0, "There must be assemblies already loaded in the process");
+                AppDomain.CurrentDomain.Load(typeof(AppDomainTests).Assembly.GetName().FullName);
+                Assembly[] assemblies1 = AppDomain.CurrentDomain.GetAssemblies();
+                // Another thread could have loaded an assembly hence not checking for equality
+                Assert.True(assemblies1.Length >= assemblies.Length, "Assembly.Load of an already loaded assembly should not cause another load");
+                Assembly.LoadFile(typeof(AppDomain).Assembly.Location);
+                Assembly[] assemblies2 = AppDomain.CurrentDomain.GetAssemblies();
+                Assert.True(assemblies2.Length > assemblies.Length, "Assembly.LoadFile should cause an increase in GetAssemblies list");
+                int ctr = 0;
+                foreach (var a in assemblies2)
                 {
-                    if (a.Location == typeof(AppDomain).Assembly.Location)
-                        ctr++;
+                    // Dynamic assemblies do not support Location property.
+                    if (!a.IsDynamic)
+                    {
+                        if (a.Location == typeof(AppDomain).Assembly.Location)
+                            ctr++;
+                    }
                 }
-            }
-            foreach (var a in assemblies)
-            {
-                if (!a.IsDynamic)
+                foreach (var a in assemblies)
                 {
-                    if (a.Location == typeof(AppDomain).Assembly.Location)
-                        ctr--;
+                    if (!a.IsDynamic)
+                    {
+                        if (a.Location == typeof(AppDomain).Assembly.Location)
+                            ctr--;
+                    }
                 }
-            }
-            Assert.True(ctr > 0, "Assembly.LoadFile should cause file to be loaded again");
+                Assert.True(ctr > 0, "Assembly.LoadFile should cause file to be loaded again");
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [Fact]
         [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Does not support Assembly.LoadFile")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
@ViktorHofer desktop seems to be doing that by design. It was disabled before without any reason and is still disabled without any reason provided. I do not consider this that much related to the purpose of this PR so I only went with quick fixes.

I think we should do large sweep in the end for all disabled issues without any reason and assign this work items to owners which can likely assess this much faster